test/login-mutation-test.ts

```diff
@@ -58,9 +59,13 @@ describe("Login-mutation-test", () => {
       endpoint,
       buildLoginMutationInput(userInput.email, userInput.password),
     );
+    const token = jwt.sign(
+      { userId: userResponse.id },
+      process.env.TOKEN_JWT as Secret | "",
+    );
     const loginResponseExpected = {
       user: userResponse,
-      token: " ",
+      token: token,
```

Comment:
coisinha legal do Javascripto: se a chave tem o mesmo nome da variável, dá pra omitir
```suggestion
      token,
```